Bottom: 6ebcf1ca7cf1644ebe8abc44e7ab4748d9d009f4
Top:    6f7056e2eec6422cd742f8e916459e7a012cde5a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:57 -0500

Fix printf in docker_build

Printf statement was expanding a variable into the format string,
which caused problems when that variable contained patterns that
appeared to be invalid format specifiers.


---
diff --git a/ansible-playbook/library/docker_build b/ansible-playbook/library/docker_build
index 8cc11ed..da0c78c 100644
--- a/ansible-playbook/library/docker_build
+++ b/ansible-playbook/library/docker_build
@@ -42,7 +42,7 @@ build_result="${?}"
 
 if [[ ${build_result} != "0" ]]; then
     # build failed
-    printf "{\"failed\": true, \"changed\": true, \"stdout\": \"${build_log}\"}"
+    printf "{\"failed\": true, \"changed\": true, \"stdout\": \"%s\"}" "${build_log}"
     exit 1
 fi
